src/app.py

```diff
@@ -14,12 +16,13 @@
 
 app = FastAPI(
     openapi_url="/documentation/json",
-    docs_url=None,
+    # docs_url=None,
```

Comment:
Can we remove this commented line, or it has been left here on purpose?

Reply:
It was commented out to enable the internal microservice swagger in order to perform some tests. I removed the comment

Reply:
ok thanks, btw I still see the comment, maybe the last commit is missing

Reply:
I forgot to push. Solved!